assets/javascripts/kitten/components/modals/modal.js

```diff
@@ -34,6 +45,18 @@ export class Modal extends Component {
     )
   }
 
+  renderTriggerAction(trigger) {
```

Comment:
Plutôt que de passer `trigger`, ce n'est pas mieux de faire `this.props.trigger` ici ?